mne/connectivity/cfc.py

```diff
@@ -0,0 +1,393 @@
+    raw_spec_amplitude : ndarray (n_freqs, n_times)
+        Time-frequency amplitude traces for amplitude modulated frequencies.
+    evoked_spec_amplitude : ndarray, shape (n_freqs, n_times_epoch)
+        The average evoked spectral amplitude before/after phase peaks. This
+        will have a peri-peak time window where tmin < times < tmax.
+    evoked_raw : array, shape (n_times_epoch,)
+        Average (or evoked) raw signal before/after phase peaks. This
+        will have a peri-peak time window where tmin < times < tmax.
+    trigger_inds : array (trigger_inds,)
+        Indices of maxima in the phase signal.
+
+    References
+    ----------
+    [1] High gamma power is phase-locked to theta oscillations in human
+    neocortex. Canolty RT1, Edwards E, Dalal SS, Soltani M, Nagarajan SS,
+    Kirsch HE, Berger MS, Barbaro NM, Knight RT.
+    (Science. 2006)
+    """
+
+    if data.ndim != 1:
+        raise ValueError('Dimensions of data incorrect. Please use 1d array.')
+
+    Ws = morlet(sfreq, [freq_phase], n_cycles=n_cycles,
+                sigma=None, zero_mean=True)
+    data = data.reshape([1, -1])
+    x_low = cwt(data, Ws, use_fft=False).ravel()
+
+    # Calculate phase of low frequency and find phase peaks for time-locking
+    phases = np.angle(x_low)
+    trigger_inds, _ = peak_finder(phases)
+
+    n_times = data.shape[1]
+    n_freqs = len(freqs_amplitude)
+
+    if tmin > 0 or tmax < 0:
+        raise ValueError('tmin must be < 0 and tmax must be > 0')
+
+    # obtain min and max indices around phase peaks
+    n_times_min = np.abs(int(round(tmin * sfreq)))
+    n_times_max = np.abs(int(round(tmax * sfreq)))
+    times = np.arange(-n_times_min, n_times_max, dtype=np.float) / sfreq
+
+    # makes sure all triggers have enough time bef/after phase peak
+    trigger_inds = trigger_inds[trigger_inds > n_times_min]
+    trigger_inds = trigger_inds[trigger_inds < n_times - n_times_max]
+
+    # Compute amplitude for amplitude modulated frequencies
+    Ws = morlet(sfreq, freqs_amplitude, n_cycles=n_cycles, sigma=None,
+                zero_mean=True)
+
+    parallel, my_abs_cwt, n_jobs = parallel_func(_abs_cwt, n_jobs)
+
+    if n_jobs == 1:
+        raw_spec_amplitude = np.empty((n_freqs, n_times), dtype=np.float)
+        for e, W in enumerate(Ws):
+            raw_spec_amplitude[e] = np.array(_abs_cwt(data, W))
+    else:
+        raw_spec_amplitude = np.array(
+            parallel(my_abs_cwt(data, W) for W in Ws))
+
+    # zscore for normalization
+    raw_spec_amplitude -= np.mean(raw_spec_amplitude, axis=1)[:, None]
+    raw_spec_amplitude /= np.std(raw_spec_amplitude, axis=1)[:, None]
+
+    # phase-triggered average of raw signal:
+    # and phase-triggered time-frequency amplitude values (normalized):
+    evoked_raw = np.zeros(len(times))
+    evoked_spec_amplitude = np.zeros((n_freqs, len(times)), dtype=np.float)
+    for ind in trigger_inds:
+        evoked_raw += data[0, ind - n_times_min: ind + n_times_max]
```

Comment:
Here is another place that will probably just need to be `ind + n_times_min` once you get rid of the `abs` calls above. It might be worth renaming the variables `min_offset` or `min_idx` or something, since it's no longer a number but an index offset.